check_lang.py

```diff
@@ -0,0 +1,20 @@
+from dotenv import load_dotenv
+import os
+from deep_translator import GoogleTranslator
+import langdetect
+load_dotenv()
+
+auto_translate = os.environ.get("AUTO_TRANSLATE")
+
+def translate(text):
```

Comment:
Personally I'd remove the translation feature from this PR so that the great core improvements can be viewed separately since this feature might be a bit controversial (requires internet, uses Google Translate,..). 
If you decide to leave them in though it would be awesome if you could mention it in the README and add `AUTO_TRANSLATE` to the example.env

Reply:
To be honest all the changes I have made to in this PR are changes I had to make to make privateGPT functional for me. And so I thought that just as they are useful for me they might be useful for someone else. In any case, actually the translation function as useful as it is, it totally goes against the purpose of this project, so yes, I will remove it.
I was thinking, however, of having the model locally translate the prompt

For example asking Vicuna "If this text is not English, translate it into English." After Vicuna does the translation you use Vicuna's response to execute the prompt. This would ensure privacy